### ISSUE TITLE
Is this repo dead?

### ISSUE THREAD

**Issue by imrichardcole** [twxs/vs.language.cmake#107]
I've raised a PR some time ago to add more syntax highlighting and it's not been checked.

**Comment by imrichardcole** [comment]
All - I now have collaborator access.  Please may I ask that you link your issues into this one so I can review the priority in the backlog.

Thanks

Richard

**Comment by kenwarner** [comment]
so I guess it's dead?

**Comment by imrichardcole** [comment]
@kenwarner I think so.  I've tried all methods of communication with the original owner.  I was in a position to make changes, but without the publish keys to actually be able to publish new versions to marketplace, it wasn't much use.

**Comment by twxs** [comment]
see https://github.com/twxs/vs.language.cmake/issues/119

**Comment by computerquip-work** [comment]
Can someone explain the situation as of current? There doesn't seem to be an explicit mention of where everything stands. Is this project now completely deprecated with Microsoft's CMake tools extension?

**Comment by twxs** [comment]
Yes, this extension is deprecated since Microsoft include no all the features in their extension
